src/component/header/styles/header.js

```diff
@@ -1,55 +1,217 @@
 import styled from "styled-components";
+import { Link as ReactRouterLink } from "react-router-dom";
 
-export const Container = styled.div``
-export const ButtonLink = styled.a`
-    font-style: normal;
-    font-weight: 700;
-    font-size: 14px;
-    line-height: 24px;
+export const Container = styled.div`
+  margin-top: 30px;
+  width: 100%;
+  position: static;
+  transition: all 0.4s ease;
 
-    text-align: center;
-    letter-spacing: 0.2px;
+  @media (max-width: 1000px) {
+    position: fixed;
+    top: 0;
+    transform: ${({ background }) =>
+      background ? "none" : "translate3d(0, -100%, 0)"};
+    opacity: ${({ background }) => (background ? "1" : "0")};
+    height: ${({ background }) => (background ? "100%" : "0")};
+    height: 80px;
+    margin-top: 0;
+    box-shadow: 0px 4px 49px rgba(0, 0, 0, 0.13);
```

Comment:
add background color without that its background is transparent

Reply:
okay I saw that making changes